toolchain/sem_ir/file.h

```diff
@@ -93,6 +93,11 @@ class File : public Printable<File> {
   }
   auto OutputYaml(bool include_singletons) const -> Yaml::OutputMapping;
 
+  // Returns the set of all insts corresponding to expressions that are used
+  // in positions where a `ref` tag is needed. Should only be called if
+  // has_errors is false.
```

Comment:
I think it'd be good to mention here that this is only for validation, and should only be called if `!NDEBUG` because it does a complete walk over the IR.